sdk/src/beta9/sync.py

```diff
@@ -63,6 +81,12 @@ def get_workspace_object_id() -> str:
 .ruff_cache
 .dockerignore
 .ipynb_checkpoints
+.env.local
+.envrc
+**/__pycache__/
+**/.pytest_cache/
+**/node_modules/
+*.pyc
```

Comment:
If we negate, do we even need any rules after the last rule on line 66?

Reply:
I left these here because I wasn't sure what would happen if some of these folders contained allowed file types. For instance, .vscode will have some json in it